TESTS/mbed_drivers/lp_timeout/main.cpp

```diff
@@ -120,29 +300,48 @@ void lp_timeout_500us(void)
 
 }
 
-utest::v1::status_t greentea_failure_handler(const Case *const source, const failure_t reason) {
+utest::v1::status_t greentea_failure_handler(const Case * const source, const failure_t reason)
+{
     greentea_case_failure_abort_handler(source, reason);
     return STATUS_CONTINUE;
 }
 
 Case cases[] = {
+    Case("Test callback called once (with attach)", test_callback_fires_once<TimeoutAttachTester>),
+    Case("Test callback called once (with attach_us)", test_callback_fires_once<TimeoutAttachUSTester>),
+
+    Case("Test callback not called when cancelled (with attach)", test_cancel<TimeoutAttachTester>),
+    Case("Test callback not called when cancelled (with attach_us)", test_cancel<TimeoutAttachUSTester>),
+
+    Case("Test callback override (with attach)", test_override<TimeoutAttachTester>),
+    Case("Test callback override (with attach_us)", test_override<TimeoutAttachUSTester>),
+
+    Case("Test multiple timeouts running in parallel (with attach)", test_multiple<TimeoutAttachTester>),
+    Case("Test multiple timeouts running in parallel (with attach_us)", test_multiple<TimeoutAttachUSTester>),
+
+    Case("Test zero delay (with attach)", test_no_wait<TimeoutAttachTester>),
+    Case("Test zero delay (with attach_us)", test_no_wait<TimeoutAttachUSTester>),
```

Comment:
These tests fail on _NUCLEO_F070RB_, _NUCLEO_F429ZI_ and _NUCLEO_L476RG_ -- issue #5159.